### ISSUE TITLE
评论优化建议

### ISSUE THREAD

**Issue by epubreader** [wmui/essay#45]
评论优化建议：不支持对评论进行回复

**Comment by jimmyvanelsycgz-cmyk** [comment]
Sí, al probar la demo de Gates of Olympus, puedes activar los giros gratis igual que en el juego real [https://the-gates-of-olympus.net/mx/gates-of-olympus-demo/](https://the-gates-of-olympus.net/mx/gates-of-olympus-demo/). Solo necesitas obtener 4 o más símbolos Scatter. Durante la ronda, los multiplicadores no se reinician y se van acumulando, lo que puede generar premios virtuales enormes. Es una excelente manera de conocer las fases más emocionantes del juego sin gastar ni un centavo.

**Comment by zhenmang** [comment]
您好，您的邮件我已收到
